calcom/cal.com
# feat: The 'confirm' button triggers after entering the code

## What does this PR do?

Users no longer need to click on `confirm` after entering the 6-digit code for bookings with booker email confirmation required event-types.

Fixes #12127 

## Type of change




## How should this be tested?

1. Go to booking page of an event-type that has booker email verification turned on. Select a time slot and click on verify. Once you enter the 6-digit code, you'll no longer need to click on `confirm`.

## Mandatory Tasks
